python/lsst/obs/pfs/pfsMapper.py

```diff
@@ -365,7 +368,7 @@ def std_raw(self, item, dataId):
             else:
                 var = exp.image.array/gain
 
-            exp.variance.array[:] = var
+            exp.variance.array[:] = var + channel.getReadNoise()**2
```

Comment:
Are these the correct units? I believe the CCDs come out of ISR with units of electrons. I expect the readnoise here should be in electrons, but is the NIR image array also electrons?